### ISSUE TITLE
What are these errors?

### ISSUE THREAD

**Issue by ethereal94** [katzer/cordova-plugin-local-notifications#1974]
After installing the plugin everything stops working and waterfall errors!

Who knows what the problem is and how to solve it?
Thank you!

Image:
https://postimg.cc/f36DwfGG
https://postimg.cc/8FFTQ00d
https://postimg.cc/gxtG7WNk
https://postimg.cc/Hrkp3x0h

**Comment by salteggs** [comment]
Hey, we ran into the same problem and ended up installing AndroidX and modifying the code.

You will want to change from `android.support.v4.app.NotificationCompat` to `androidx.core.app.NotificationCompat`, from `android.support.v4.util.ArraySet` to `androidx.collection.ArraySet` and so on.

We have a small install base, and since release, this plugin is now causing 0,5% crashes. We will see if we can identify the problem, maybe we did not modify the java code correctly in a specific file.
